frb_codegen/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 categories.workspace = true
 description.workspace = true
 edition.workspace = true
-rust-version = "1.74.0"
```

Comment:
Well after merging I realize, we should not bump this rust-version, otherwise users cannot use it with old Rust!

Could you please make a new PR to lower this value?

Reply:
I have just submitted PR #2458 :)